python/LICENSE

```diff
@@ -1,11 +1,7 @@
 Copyright (c) 2022 AICA SA
 
-Permission is hereby granted, free of charge, to any person obtaining a copy
-of this software and associated documentation files (the "Software"), to deal
-in the Software without restriction, including without limitation the rights
-to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
-copies of the Software, and to permit persons to whom the Software is
-furnished to do so, subject to the following conditions:
+
+
```

Comment:
Why this change? If we remove it then we should only leave one empty line

Reply:
Yes I did, I would keep this text in that case thanks